assets/js/modules/adsense/datastore/urlchannels.js

```diff
@@ -94,7 +93,19 @@ export const actions = {
 };
 
 export const controls = {
-	[ FETCH_URLCHANNELS ]: ( { payload: { accountID, clientID } } ) => {
+	[ FETCH_URLCHANNELS ]: ( { payload: { clientID } } ) => {
+		const accountID = parseAccountID( clientID );
+		if ( 'undefined' === typeof accountID ) {
+			// Mirror the API response that would happen for an invalid client ID.
```

Comment:
Why do this locally if the server will throw this sort of response? I don't think the check is incorrect, but I don't know why we'd re-implement it on the client side if the API already handles this.

Reply:
The server requires passing the `accountID` parameter, so we need to parse that on the client. If parsing fails, we know the `clientID` is invalid, and I just wanted to mirror the same response the server would provide for that case.

Also this avoids issuing the server request unnecessarily, since we'd already know here that it will fail.

Reply:
Ah, okay, that makes sense. 👍